packages/eddsa-poseidon/src/blake.ts

```diff
@@ -156,7 +157,7 @@ function lengthCarry(arr: number[]) {
  * hashing, allowing data to be added in chunks.
  */
 /* eslint-disable import/prefer-default-export */
-export class Blake512 {
+export class Blake512 implements HashFunction {
```

Comment:
Let's make `Blake512` and `Blake2b` class exports consistent. Like making this the default import.

Reply:
done!